src/thirdparty/openapi.ts

```diff
@@ -681,23 +718,24 @@ export interface components {
       middleName?: components['schemas']['MiddleName'];
       lastName?: components['schemas']['LastName'];
     };
+    /**
+     * Identifier of the Party.
+     */
+    PartyIdentifier: string;
     /**
      * Data model for the complex type PartyIdInfo.
      */
     PartyIdInfo: {
       partyIdType: components['schemas']['PartyIdType'];
-      /**
-       * Identifier of the Party.
-       */
-      partyIdentifier: string;
-      /**
-       * Either a sub-identifier of a PartyIdentifier, or a sub-type of the
-       * PartyIdType, normally a PersonalIdentifierType.
-       */
-      partySubIdOrType?: string;
+      partyIdentifier: components['schemas']['PartyIdentifier'];
```

Comment:
nit: please leave in the nice comments we had above these fields

Reply:
this file is generated when building, it seems it just moves the comments around https://github.com/mojaloop/api-snippets/pull/46/files/2f43eee51f649905b037eb61029d7634d056af93#diff-978850faba52906f4cc439ebc467fdc887055871fe18358001ec0c3c0af17a79R807

Reply:
ok cool, good to know